sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-keyvault-certificates-server-side/script/export_environment_variables.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Set your Azure Subscription id where all required resources will be created.
+export SUBSCRIPTION_ID=799c12ba-353c-44a1-883d-84808ebb2216
```

Comment:
Make it like a fake one, so use will not try to use it. For example `00000000-0000-0000-0000-0000000000000000`

Reply:
Sorry I forgot to delete them. I think leaving it empty is ok? just like 
```
# Set your Azure Subscription id where all required resources will be created.
export SUBSCRIPTION_ID=

# Set the name for your Azure resource group to be created.
export RESOURCE_GROUP_NAME=
```